tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Persistence/Repositories/StylesheetRepositoryTest.cs

```diff
@@ -300,8 +300,8 @@ public void PathTests()
                 Assert.AreEqual("path-2\\test-path-3.css".Replace("\\", $"{Path.DirectorySeparatorChar}"), stylesheet.Path);
                 Assert.AreEqual("/css/path-2/test-path-3.css", stylesheet.VirtualPath);
 
-                stylesheet = new Stylesheet("\\test-path-4.css") { Content = "body { color:#000; } .bold {font-weight:bold;}" };
-                Assert.Throws<UnauthorizedAccessException>(() => // fixed in 7.3 - 7.2.8 used to strip the \
+                stylesheet = new Stylesheet("..\\test-path-4.css") { Content = "body { color:#000; } .bold {font-weight:bold;}" };
```

Comment:
Im not 100% sure about these changes.. Are this not considered a breaking change if this is required?

Reply:
It is relaxing the file path check a little bit (which isn't breaking existing things, as it wouldn't have worked previously) and making it align more with how the `PhysicalFileProvider` handles paths (always relative/rooted to the base path).

The `PhysicalFileSystem` implementation already checks whether paths are within the root path/not outside the file system, so a leading (back)slash can be safely treated as being relative to the root path instead of throwing exceptions.

Reply:
I've added some extra tests to validate saving and getting the rooted paths (starting with a slash) do actually work 👍🏻